src/main/webapp/resources/js/pages/UserGroupsPage/components/CreateNewUserGroupButton.jsx

```diff
@@ -3,13 +3,14 @@ import { AddNewButton } from "../../../components/Buttons/AddNewButton";
 import { Form, Input, Modal } from "antd";
 import { createUserGroup } from "../../../apis/users/groups";
 import { useNavigate } from "@reach/router";
+import { setBaseUrl } from "../../../utilities/url-utilities";
```

Comment:
Consider use `useResetFormOnCloseModal` it automatically resets the form when the modal closes.  See `AddGroupButton.jsx`.